tasks/docs.rb

```diff
@@ -305,7 +305,7 @@ def build_resource_data(name, data)
       else
         puts "Writing out #{resource}."
         FileUtils.mkdir_p "docs_site/#{resource}"
-        File.open("docs_site/#{resource}/_index.md", "w") { |f| f.write(resource_data.to_yaml) }
+        File.open("docs_site/#{resource}/_index.md", "w") { |f| f.write(resource_data.to_yaml + "---") }
```

Comment:
Nitpick, but a comment about why the `"---"` got added would be good so it doesn't get left off in the future.